wolfSSL/src/appendix07.md

```diff
@@ -31,8 +31,6 @@ These use different kinds of math from the conventional algorithms. They are des
 * Dilithium Signature Scheme
 * FALCON Signature Scheme
 * KYBER KEM
-* SABER KEM
```

Comment:
Please add a note that SBER and NTRU were removed because they are not going to be standardized in the current NIST finalization / round.